File: Java/Car.java
class Car {
    // Attributes
    Integer id;
    String license;
    Account driver;
    Integer passenger;

    // Constructor
    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    // Methods
    void printDataCar(){
        System.out.println("License: " + license + "Name driver: "+ driver.name);
    }
}

File: PHP/Car.php
<?php

require_once('./Account.php');

class Car 
{
    public $id;
    public $license;
    public $driver;
    public $passengers;

    public function __construct($license, $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function PrintDataCar(){
        print 'License: '.$this->license;
        print 'Driver: '.$this->driver->name;
    }
}
?>